### DISCUSSION TITLE
A reference database download link might be outdated or not working

### DISCUSSION THREAD

**Discussion by h171e** [PGScatalog/pgsc_calc#213] [Q&A] [answered]
Noticed that the link referenced in the optional step **3. (Optional) Download reference database** located in the **Getting started** page of the read the docs website `https://pgsc-calc.readthedocs.io/en/latest/getting-started.html` might be referencing an inexistent file or folder.
 
` wget https://ftp.ebi.ac.uk/pub/databases/spot/pgs/resources/pgsc_calc.tar.zst`

Checking the directory at `https://ftp.ebi.ac.uk/pub/databases/spot/pgs/resources/` made me notice two files that make me wonder if they are required or used by the pipeline with the `--run_ancestry` parameter and might be needed to be located in the same folder as the reference file. 

**GRCh38_HAPNEST_reference.tar.zst**
`https://ftp.ebi.ac.uk/pub/databases/spot/pgs/resources/GRCh38_HAPNEST_reference.tar.zst`
**GRCh38_HAPNEST_target.tar.zst**
`https://ftp.ebi.ac.uk/pub/databases/spot/pgs/resources/GRCh38_HAPNEST_target.tar.zst`

**Comment by nebfield** [comment, marked as answer]
Thanks for letting us know! we added HGDP to the reference panel and it looks like some old links are still hiding in the docs.

The best database to use is [the combined panel](https://ftp.ebi.ac.uk/pub/databases/spot/pgs/resources/pgsc_HGDP+1kGP_v1.tar.zst), but it's quite big so we [also provide only 1000 Genomes](https://ftp.ebi.ac.uk/pub/databases/spot/pgs/resources/pgsc_1000G_v1.tar.zst). `--run_ancestry` only supports one database path. 

The HAPNEST databases are smaller synthetic datasets that we use to run automated tests on GitHub Actions and aren't useful when working with real biological data.

I'll dig through the docs and update any old links.
